spec/services/activities/import_from_csv_spec.rb

```diff
@@ -162,53 +164,41 @@
     end
 
     it "has an error and does not update any other activities if an Activity does not exist" do
+      invalid_activity_attributes = {
+        "RODA ID" => "FAKE RODA ID",
+        "Title" => "Here is another title",
+        "Description" => "Another description goes here...",
+        "Recipient Region" => "789",
+      }
       activities = [
         existing_activity_attributes,
-        {
-          "RODA ID" => "FAKE RODA ID",
-          "Title" => "Here is another title",
-          "Description" => "Another description goes here...",
-          "Recipient Region" => "789",
-        },
+        invalid_activity_attributes,
       ]
 
       expect { subject.import(activities) }.to_not change { existing_activity }
 
       expect(subject.created.count).to eq(0)
       expect(subject.updated.count).to eq(0)
-
-      expect(subject.errors.count).to eq(1)
```

Comment:
As the `it` block starts with "has an error", we should probably leave the expectation to check that there is an error in, even if we're no longer checking for an exact error message.